### PR TITLE
Better better logging

### PR DESCRIPTION
**Pull request by tomchop**
Add a RotatingFileHandler to the modules logger.

### CODE REVIEW

File: dftimewolf/lib/module.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 import abc
 import logging
+from logging import handlers
```
Comment (Onager) [inline dftimewolf/lib/module.py:6]: Why import this again? Just do logging.handlers.

File: dftimewolf/cli/dftimewolf_recipes.py [hunk 2]
```diff
@@ -254,15 +253,16 @@ def SetupLogging():
   # File handler needs go be added first because it doesn't format messages
   # with color
   file_handler = handlers.RotatingFileHandler(
-      DEFAULT_LOG_FILE, maxBytes=5*1024*1024, backupCount=3)
+      logging_utils.DEFAULT_LOG_FILE, maxBytes=5*1024*1024, backupCount=3)
```
Comment (Onager) [inline dftimewolf/cli/dftimewolf_recipes.py:256]: Move the maxbytes and backupCount values here to logging_utils as well

File: dftimewolf/lib/module.py [hunk 3]
```diff
@@ -30,10 +31,21 @@ def __init__(self, state, critical=False):
...
     self.critical = critical
     self.state = state
+    self.SetupLogging()
+
+  def SetupLogging(self):
+    """Sets up stream and file logging for a specific module."""
     self.logger = logging.getLogger(name=self.__class__.__name__)
+
+    file_handler = handlers.RotatingFileHandler(
+        logging_utils.DEFAULT_LOG_FILE, maxBytes=5*1024*1024, backupCount=3)
```
Comment (Onager) [inline dftimewolf/lib/module.py:41]: Move the maxbytes and backupCount values here to logging_utils as well

File: dftimewolf/lib/module.py (hunk 1 above)
Comment (tomchop) [reply dftimewolf/lib/module.py:6]: I'd love to, but doing so caused some actual runtime errors: https://github.com/log2timeline/dftimewolf/pull/306

Comment (Onager) [reply dftimewolf/lib/module.py:6]: Hmm, that's weird - please add a comment explaining that

**Pull request merged**